pkg/rpcclient/actor/actor.go

```diff
@@ -45,7 +44,8 @@ type SignerAccount struct {
 // Actor keeps a connection to the RPC endpoint and allows to perform
 // state-changing actions (via transactions that can also be created without
 // sending them to the network) on behalf of a set of signers. It also provides
-// an Invoker interface to perform test calls with the same set of signers.
+// an Invoker interface to perform test calls with the same set of signers. It
+// also provides a Waiter interface to perform transaction acceptance awaiting.
```

Comment:
We need a better documentation on RPC->Waiter rules.

Reply:
Fixed.